runtime/Cargo.toml

```diff
@@ -82,6 +82,7 @@ sys-info = "0.9.1"
 termcolor = "1.1.3"
 tokio = { version = "1.17", features = ["full"] }
 uuid = { version = "1.0.0", features = ["v4"] }
+portable-pty = "0.7.0"
```

Comment:
I wonder if we could instead refactor and reuse the code in test_util/src/pty.rs

Reply:
> I wonder if we could instead refactor and reuse the code in test_util/src/pty.rs

Maybe, but it needs to be async on the input and output, so some tokio magic would have to be used. I'll take a look and see what I find out!

Reply:
Seems doable, I'm gonna try to get some work done on it, but most likely won't really get to it until next week.